src/main/java/seedu/address/model/car/Model.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.car;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Car's Model in MATER.
+ * Guarantees: immutable; is valid as declared in {@link #isValidModel(String)}
+ */
+public class Model {
```

Comment:
I think Model Class needs to be renamed as other parts of the codebase use Model in the Programming sense of the word and it will be confusing

Reply:
Renamed Model to CarModel, Make to CarMake.